pkg/redisnode/config.go

```diff
@@ -20,20 +20,23 @@ const (
 	HTTPServerAddrDefault = "0.0.0.0:8080"
 	// RedisMaxMemoryDefault default redis max memory
 	RedisMaxMemoryDefault = 0
+	// RedisMaxMemoryDefaultPolicy default redis max memory evition policy
+	RedisMaxMemoryDefaultPolicy = "noeviction"
```

Comment:
should be name: `RedisMaxMemoryPolicyDefault`

Reply:
Good spotting :)